django/test/runner.py

```diff
@@ -633,7 +633,8 @@ def get_test_runner_kwargs(self):
     def run_checks(self):
         # Checks are run after database creation since some checks require
         # database access.
-        call_command('check', verbosity=self.verbosity)
+        #call_command('check', verbosity=self.verbosity)
```

Comment:
FWIW this will likely not be necessary if https://github.com/django/django/pull/12201 lands as it will allow passing `databases=self.databases` to the command and disable running checks on unrelated ones.

Reply:
Yeah, this was just a temporary hack to at least make things work until that PR is merged.